segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLog.java

```diff
@@ -354,6 +354,11 @@ public QueueStats getQueueStatistics() {
      * Write Processor main loop. This method is not thread safe and should only be invoked as part of the Write Processor.
      */
     private void processWritesSync() {
```

Comment:
To confirm, even though there is a change in the internal behavior of this method, there is already enough test cases covering this code path? I'm trying to understand whether we need more testing.

Reply:
There are enough tests for this class.